packages/web-components/src/radio-group/radio-group.styles.ts

```diff
@@ -0,0 +1,63 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  colorCompoundBrandForeground1,
+  colorNeutralForeground1,
+  colorNeutralForeground3,
+  colorNeutralStrokeAccessible,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingHorizontalXS,
+  spacingVerticalS,
+  spacingVerticalSNudge,
+} from '../theme/design-tokens.js';
+
+/** RadioGroup styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  :host {
+    --control-border-color: ${colorNeutralStrokeAccessible};
+    --checked-indicator-background-color: ${colorCompoundBrandForeground1};
+    --state-color: ${colorNeutralForeground3};
+    --stacked-padding: ${spacingVerticalSNudge} ${spacingHorizontalS} ${spacingVerticalSNudge} ${spacingHorizontalXS};
+    align-items: flex-start;
+    flex-direction: column;
+    margin: 2px 0;
```

Comment:
@brianchristopherbrady Should the *host* of the component have 2px of margin? That kind of violates that components care about _their_ presentation. Any value in a contextual scenario which is different needs to reassign this. It seems like maybe this is intended to be somewhere else?